test/channels/application_cable/connection_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'test_helper'
```

Comment:
- [ ] Let's make sure you have required tests for all the important functions as requested in project
![image](https://user-images.githubusercontent.com/58504655/227234441-d4726611-80af-494a-bcaa-130afd381b21.png)